### DISCUSSION TITLE
npm install issue -

### DISCUSSION THREAD

**Discussion by HazardJ** [microsoft/FluidFramework#14674] [Q&A] [answered]
I'm a lawyer (!) exploring Fluid.  I have lots of experience with git and VSCode, and some experience with command line and all that but ... 
When I do the install, I hit a problem, or series of problems that look similar.  Like this:

ERROR in ./node_modules/@fluidframework/container-loader/lib/utils.js 5:0-28
Module not found: Error: Can't resolve 'url' in '/Users/jgh/Documents/GitHub/fluid/FluidHelloWorld/node_modules/@fluidframework/container-loader/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

It is very likely that I had very old installs of npm and related.  Might that be the problem?
I couldn't figure out what is the next step in debugging. 

Thanks, Jim

ps:  GitHub.com/CommonAccord/

**Comment by ChumpChief** [comment, marked as answer]
The FluidHelloWorld repo was updated to use Webpack 5 after this discussion was posted (https://github.com/microsoft/FluidHelloWorld/pull/301), so I suspect this issue has been fixed.
